basicChecks.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+// These should pass for *almost* every rule
+module.exports = [
+	{
+		code: '',
+		description: 'empty stylesheet',
+	},
+	{
+		code: 'a {}',
+		description: 'empty rule',
+	},
+	{
+		code: '@import "foo.css";',
+		description: 'blockless statement',
+	},
+	{
+		code: ':global {}',
+		description: 'CSS Modules global empty rule set',
+	},
+];
```

Comment:
[note] This is a copy from `lib/testUtils/basicChecks.js` in the `stylelint/stylelint` repo.

https://github.com/stylelint/stylelint/blob/42ac6a190c7fdbcc01b35f0642a4111e647c0389/lib/testUtils/basicChecks.js

Within the repo above, `basicChecks.js` seems unused.